rdr_service/model/participant_summary.py

```diff
@@ -1555,8 +1557,23 @@ def enrollmentSiteId(cls):
     """User who recorded ID verification occurrence"""
     onsiteIdVerificationSite = Column("onsite_id_verification_site_id", Integer, ForeignKey("site.site_id"))
     """The site where ID verification took place"""
+
+    remoteIdVerificationOrigin = Column("remote_id_verification_origin", String(80))
+    """ The sign up portal where ID Verification took place ( Vibrent OR Care Evolution ) """
+
+    remoteIdVerificationStatus = Column(
+        "remote_id_verification_status",
+        Enum(RemoteIdVerificationStatus),
+        default=RemoteIdVerificationStatus.FALSE
+    )
+    """ A true or false value that indicates whether the ID Verification is Remote  """
+
+    remoteIdVerifiedOn = Column("remote_id_verified_on", Date)
```

Comment:
Think this should be `UTCDateTime`, you will need to edit the migration as well.

Reply:
Maybe we can simplify this and just use remoteIdVerificationStatus = Column("remote_id_verification_status", Boolean, default=0) instead of leveraging another enum

Reply:
Still needs to be changed to `UTCDateTime`